### PR TITLE
Session fix

### PR DESCRIPTION
**Pull request by mrvanes**
Fixes SimpleSAML_Session::getInstance() deprecation in simpleSAMLphp 1.14

### CODE REVIEW

File: www/dashboard.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 require __DIR__ . '/_includes.php';
 
 set_time_limit(180);
-$session = SimpleSAML_Session::getInstance();
```
Comment (thijskh) [inline www/dashboard.php]: $session is not used lateron?

File: www/importentity.php [hunk 2]
```diff
@@ -228,6 +230,7 @@
 
 function janus_array_diff_recursive($array1, $array2)
 {
+    if (!is_array($array2)) return $array1;
```
Comment (thijskh) [inline www/importentity.php]: This is actually pr #599

File: www/dashboard.php (hunk 1 above)
Comment (mrvanes) [inline www/dashboard.php]: Correct, I checked everything. It can safely be removed.

Comment (thijskh) [inline www/dashboard.php]: Then why is it still added one line below this one? :)

Comment (thijskh) [inline www/dashboard.php]: :arrow_down:

Comment (mrvanes) [inline www/dashboard.php]: Please read all the changes. I changed many files to the best of my knowledge and checked every files for (re)use of $session. If I removed it, it wasn't used. If I changed it to getSession it was needed later.

File: www/importentity.php (hunk 2 above)
Comment (mrvanes) [inline www/importentity.php]: Yes, I have troubles getting pull request clean from all other fixes I came up with. I thought I'd managed to keep it out, but apparently I didn't.

**Pull request merged**